documentation/get-started/contribute.en..md

```diff
@@ -0,0 +1,31 @@
+---
+title: Contribute
+---
+
+# Contribute
+
+QField is an open source project. It is free to share, use and modify and it will stay like that. We are very happy if this app helps you in whatever creative way you may use it. If you found it useful, we will be even happier if you could give something back. A couple of things you can do are:
+
+*   Rate the app [★★★★★](https://play.google.com/store/apps/details?id=ch.opengis.qfield&hl=en#details-reviews);
+*   [Tweet about it](https://twitter.com/share?text=Looking%20for%20a%20good%20tool%20for%20field%20work%20in%20GIS?%20Check%20out%20%23QField!);
+*   Write about your experience ([please let us know!](mailto:info@opengis.ch));
+*   [Help with the documentation](https://github.com/opengisch/QField-docs#documentation-process);
+*   [Translate the documentation](https://github.com/opengisch/QField-docs#translation-process) or [the app](https://www.transifex.com/opengisch/qfield-for-qgis/);
+*   [Sponsor a feature](http://qfield.org/docs/development/index.html#make-it-grow-make-it-yours); or
+*   Just drop by to say thank you or have a beer with us next time you meet OPENGIS.ch at a conference.
+
+# Development
+
+if you want to get involved in QField development, please refer to the following doumentations:
+  - [QField repository](https://github.com/opengisch/QField/blob/master/doc/dev.md)
```

Comment:
@mbernasocchi , I'm noticing one list uses * bullet points, and this one uses - -- should it be harmonized?

Reply:
done